content/concept-amendments.md

```diff
@@ -48,7 +48,9 @@ Theoretically, a `tfLostMajority` EnableAmendment pseudo-transaction could be in
 
 Operators of `rippled` validators can choose which amendments to support or reject using the [`feature` command](reference-rippled.html#feature). This decides which amendments the validator votes for in the [amendment process](#amendment-process). By default, `rippled` votes in favor of every amendment it knows about.
 
-The operator of a `rippled` validator can "veto" an amendment. In this case, that validator never sends a vote in favor of the amendment. If enough servers veto an amendment, that prevents it from reaching consistent 80% support, so the amendment does not apply.
+The operator of a `rippled` validator can "veto" an amendment. In this case, that validator never sends a vote in favor of the amendment. If enough servers veto an amendment, that prevents it from reaching consistent 80% support, so the amendment does not apply. The amendment can still become enabled later if an 80% majority of trusted validators support the amendment, either because validator operators changed their minds or new validators that support the amendment became trusted.
+
+An amendment can gain and lose a majority an unlimited number of times before it becomes permanently enabled. As long as there are servers voting in favor of an amendment, the amendment is not permanently rejected. New versions of `rippled` can remove an amendment to ensure that they do not vote for it.
 
 As with all aspects of the consensus process, amendment votes are only taken into account by servers that trust the validators sending those votes. Ripple (the company) recommends only trusting the 5 default validators that Ripple (the company) operates. For now, trusting only those validators is enough to coordinate with Ripple (the company) on releasing new features.
```

Comment:
They're default because the default config file includes their IDs, so anyone running rippled will use those validators unless they change the config file. And since we don't recommend people change the defaults, that's how that works.